integration_tests/helpers/mergeMiningProxyProcess.js

```diff
@@ -38,7 +40,7 @@ class MergeMiningProxyProcess {
     // console.log("MergeMiningProxyProcess init - assign server GRPC:", this.grpcPort);
   }
 
-  run(cmd, args) {
+  run(cmd, args, monerodUrl) {
```

Comment:
Personally I’d rather this key/value is just added to the args object it doesn’t need to be a new function parameter

Reply:
Sorry, args is for the command line so this is fine

Reply:
Thanks for the comment, but I would rather not so that `args` can remain dedicated for the command line arguments.

Reply:
Sure I agree with that, but next time additional parameters are required then this should be refactored into an `opts` object